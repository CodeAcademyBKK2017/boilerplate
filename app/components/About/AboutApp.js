import React from 'react';
import {
  Text
} from 'react-native';
  
export default class MyNotificationsScreen extends React.Component {
    static navigationOptions = {
      tabBarLabel: 'APP'
    };
    render () {
      return (
        <Text>APP</Text>
        // <Button
        //   onPress={() => this.props.navigation.goBack()}
        //   title='Go back home'
        // />
      );
    }
}
