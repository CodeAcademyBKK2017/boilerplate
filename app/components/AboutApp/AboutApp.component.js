import Icon from 'react-native-vector-icons/FontAwesome';
import React, {Component} from 'react';
import styles from './AboutApp.style';
import Touchable from 'react-native-platform-touchable';

import {Text, View} from 'react-native';

export default class AboutAppScreen extends Component {

  static navigationOptions = ({navigation}) => {
    const toggleDrawer = (navigation) => () => navigation.navigate('DrawerToggle');
    return {
      title: 'About App',
      headerLeft: <Touchable onPress={toggleDrawer(navigation)}><Icon name='bars' size={24} /></Touchable>
    };
  }
  
  render () {
    return (
      <View style={styles.container}>
        <Text>About App</Text>
      </View>
        
    );
  }
  
}