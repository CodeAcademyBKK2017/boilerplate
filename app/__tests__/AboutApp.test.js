import 'react-native';
import AboutApp from '../components/About/AboutApp.component';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
// import {shallow} from 'enzyme';

describe('AboutApp', () => {
  it('renders correctly AboutApp', () => {
    const tree = renderer.create(
      <AboutApp />
    );
    expect(tree).toMatchSnapshot();
  });
});