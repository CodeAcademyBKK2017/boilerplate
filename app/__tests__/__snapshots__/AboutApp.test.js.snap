// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutApp renders correctly AboutApp 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    About the App
  </Text>
</View>
`;
